test/Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests/KafkaLanguageEndToEndTests.cs

```diff
@@ -16,7 +16,7 @@ public KafkaLanguageEndToEndTests(KafkaLanguageEndToEndFixture endToEndTestFixtu
             this.endToEndTestFixture = endToEndTestFixture;
         }
 
-        [Fact]
+        /*[Fact]
```

Comment:
As per my understanding and scope of this PR we are adding 2 E2E tests for Java & Python as per enhanced framework so can we remove this file as this will be depricated ??

Reply:
Removed.